_sass/color_schemes/dark.scss

```diff
@@ -1,3 +1,26 @@
+button,
+input,
+optgroup,
+select,
+option,
+textarea {
+  color: white; /* 1 */
+  font: inherit; /* 2 */
+  margin: 0; /* 3 */
+  background-color: #2c2b30;
+  padding: 10px;
+  box-shadow: 0 1px 3px rgba(0,0,0,.07), 0 4px 14px rgba(0,0,0,.05);
+  border: none;
+}
+
+input[type="text"]{
+  border: solid white;
+  border-width: thin;
+}
+
+#subjects {
+   color: white; width: 160px; height: 210px; border: 1px solid white; padding: 0 0 0 10px; float:left; background-color:#2c2b30; margin-bottom: 10px
```

Comment:
Expands to multiline to follow same convention

Reply:
Solved this @tachyons